gitops/snapshot.go

```diff
@@ -298,7 +299,13 @@ func MarkSnapshotIntegrationStatusAsInProgress(adapterClient client.Client, ctx
 		buildPipelineRunFinishTime := time.Unix(buildPipelineRunFinishTimeInt, 0)
 		buildPipelineRunFinishTimeMeta := &metav1.Time{Time: buildPipelineRunFinishTime}
 
-		go metrics.RegisterIntegrationResponse(*buildPipelineRunFinishTimeMeta, snapshotInProgressTime)
+		duration := snapshotInProgressTime.Sub(buildPipelineRunFinishTimeMeta.Time)
+		log.Info("IntegrationResponse metric",
```

Comment:
Can we have a more descriptive log message here, maybe something like "Integration service response time?"

Reply:
udpated